homeassistant/components/conversation/default_agent.py

```diff
@@ -40,14 +40,16 @@
     template,
     translation,
 )
+from homeassistant.helpers.entity_component import EntityComponent
 from homeassistant.helpers.event import (
     EventStateChangedData,
     async_track_state_added_domain,
 )
 from homeassistant.util.json import JsonObjectType, json_loads_object
 
 from .const import DEFAULT_EXPOSED_ATTRIBUTES, DOMAIN
-from .models import AbstractConversationAgent, ConversationInput, ConversationResult
+from .entity import ConversationEntity
+from .models import ConversationInput, ConversationResult
 
 _LOGGER = logging.getLogger(__name__)
 _DEFAULT_ERROR_TEXT = "Sorry, I couldn't understand that"
```

Comment:
Should this be moved to `strings.json`? This would ensure that a translated message occurs for languages that aren't yet in the intents repo.

Reply:
Let's do that in next PR. I want to limit the scope of this PR (which is already quite wide)